frontend/src/metabase/entities/questions.js

```diff
@@ -26,6 +27,62 @@ const Questions = createEntity({
   nameOne: "question",
   path: "/api/card",
 
+  /**
+   * Temporarily mock endpoints for Metrics v2
+   *
+   * Any question with `type: "metric"` will be passed to API as `type: "question"`.
+   * Same goes for any questions with a name starting with "Metric" (case-insensitive).
+   */
+  api: {
+    get: async payload => {
+      const get = GET("/api/card/:id");
+      const result = await get(payload);
+
+      if (result.name.toLowerCase().startsWith("metric")) {
+        return { ...result, type: "metric", dataset: false };
+      }
+
+      return result;
+    },
+
+    list: async payload => {
+      const get = GET("/api/card");
+      const results = await get(payload);
+
+      return results.map(result => {
+        if (result.name.toLowerCase().startsWith("metric")) {
+          return { ...result, type: "metric", dataset: false };
+        }
+
+        return result;
+      });
+    },
+
+    create: async payload => {
+      const create = POST("/api/card");
+
+      if (payload.type === "metric") {
+        const tweakedPayload = { ...payload, type: "question", dataset: false };
+        const result = await create(tweakedPayload);
+        return { ...result, type: "metric" };
+      }
+
+      return await create(payload);
+    },
+
+    update: async payload => {
+      const update = PUT("/api/card/:id");
+
+      if (payload.type === "metric") {
+        const tweakedPayload = { ...payload, type: "question", dataset: false };
+        const result = await update(tweakedPayload);
+        return { ...result, type: "metric" };
+      }
+
+      return await update(payload);
```

Comment:
idk we do we use `return await` without try catch, but seems there shouldn't be any difference comparing with just returning a promise